gradle-version/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
```

Comment:
I think I'm starting to hate these reviews because the first 11 files are always boilerplate to scroll past 😅

Reply:
I'm wondering if we can lift them up.
(I believe the editorconfig can be lifted)